src/services/completions.ts

```diff
@@ -1590,7 +1590,8 @@ namespace ts.Completions {
                 if (m.kind !== SyntaxKind.PropertyAssignment &&
                     m.kind !== SyntaxKind.ShorthandPropertyAssignment &&
                     m.kind !== SyntaxKind.BindingElement &&
-                    m.kind !== SyntaxKind.MethodDeclaration) {
+                    m.kind !== SyntaxKind.MethodDeclaration &&
+                    m.kind !== SyntaxKind.GetAccessor) {

```

Comment:
this list does not return `get` or `set`. so this can be a different feature, when i type get/set i see properties that are applicable to each. 

Now for this change, if we consider a getter to be a declaration of a property and thus remove it from the list, then i would say a setter should do the same.